www/codemirror/codemirror-index.html

```diff
@@ -407,8 +407,18 @@
           }, 25)
         }
       })
+      editor.on('change', (doc) => {
+        if (!doc.isReplacing) {
```

Comment:
this could be consolidated into 1 method but not sure what to call it 🤔
```
        if (!doc.isReplacing) {
          sendNativeMessages(doc)

          clearTimeout(cursorActivityScrollTimer)
          cursorActivityScrollTimer = setTimeout(() => {
            scrollCursorIntoViewIfNeeded()        
          }, 25)
        }
```